packages/simulators/lib/simulator_manager.dart

```diff
@@ -114,11 +114,36 @@ class IosSimulatorManager {
     //   iPhone 8 Plus (170207A8-7631-4CBE-940E-86A7815AEB2B) (Shutdown)
     //   iPhone 11 (7AEC5FB9-E08A-4F7F-8CA2-1518CE3A3E0D) (Booted)
     //   iPhone 11 Pro (D8074C8B-35A5-4DA5-9AB2-4CE738A5E5FC) (Shutdown)
+    // Example output 2 (from Mac Web Engine try bots):
+    // == Devices ==
+    // -- iOS 13.0 --
```

Comment:
Is this example demonstrating the need for the new code? If so, then I'm not sure how it does that. `restOfTheOutput.indexOf('--')` is >0 (because the string does contain `--`), so the new code won't execute 🤔

Reply:
in one of them the version was finishing with '--' now it finishes with "==". let me write an explanation, but as mentioned in the earlier comment my next PR would be just removing these.

Reply:
I have local test for these things btw which I don't merge let me also send it!